cloudslang-runtime/src/main/resources/scripts/cs_round.py

```diff
@@ -0,0 +1,2 @@
+def cs_round(val):
+  return round(val)
```

Comment:
it goes for all the rest from functions, we should probably check for the types we expect in the function, or else we'll get unpredictable results